samples/face_reid/README.md

```diff
@@ -48,24 +48,18 @@ docker compose -f docker-compose.x86.yml --profile index up
 docker compose -f docker-compose.l4t.yml --profile index up
 ```
 
-First startup can take several minutes as the module needs to convert ONNX models into TRT format. Successful module start is indicated by a log message like
+First startup can take several minutes as the `index-builder-pipeline` needs to convert ONNX models into TRT format. Successful module start is indicated by a log messages like
 
 ```
-INFO ... > The pipeline is initialized and ready to process data...
+INFO ... > The pipeline is initialized and ready to process data. Initialization took ...
+INFO ... > Setting module status to ModuleStatus.RUNNING
 ```
 
-After that the `pictures-source` container will be started automatically.
+After that the `index-builder-client` container will be started automatically.
 
-Index Builder module does not stop automatically and requires manual exit. It is safe to do so once the `face_reid-pictures-source` container exits and `face_reid-index-builder` container logs the message following messages (assuming the default gallery image set)
+The `index-builder-client` service runs the [index_builder_client.py](./src/index_builder_client.py) script, which loads the images from the [gallery](./assets/gallery), pads them to 16:9 aspect ratio if necessary, sends them to the `index-builder-pipeline` and uses the received results to create `index_files/index.bin` index file and the cropped face images in the `index_files/processed_gallery` dir.
```

Comment:
What is the reason for using a 16:9 aspect ratio (portrait usually has a different aspect ratio)? 

You can use the detector's aspect ratio and resolution.

Reply:
Changed to use the detector's resolution. Updated README.